multi-cloud/fast-start.sh

```diff
@@ -71,6 +71,8 @@ if ! GOOGLE_PROJECT=$(gcloud config get-value project 2>/dev/null) ||
   echo "gcloud config set project $latest_project" >> ~/.bashrc
 fi
 GOOGLE_PROJECT=$(gcloud config get-value project 2>/dev/null)
+echo "export GOOGLE_PROJECT=$GOOGLE_PROJECT" >> ~/.bashrc
```

Comment:
This should already be set to be `GOOGLE_CLOUD_PROJECT.` I guess this would be dummy proof?